docs/platforms/android/metrics/index.mdx

```diff
@@ -5,8 +5,6 @@ sidebar_order: 5500
 ---
```

Comment:
I would add the following to the pages' frontmatter, it removes the page from the sidebar, but makes it still viewable with direct link. This way we can avoid new users finding that page, and being confronted with the fact it's no longer meaningful for them.

```suggestion
sidebar_hidden: true
---
```